package/debian/rules.jammy

```diff
@@ -32,6 +32,7 @@ override_dh_auto_build:
 
 override_dh_auto_install:
 	package/package
+	mv $(DESTDIR)$(PREFIX)/local/lib/$(PYTHON_VERSION) $(DESTDIR)$(PREFIX)/lib/$(PYTHON_VERSION)
 	@mkdir -p $(DESTDIR)$(PREFIX)/lib/$(PYTHON_DEB_VERSION)/dist-packages
 	ln --symbolic --relative $(DESTDIR)$(PREFIX)/lib/$(PYTHON_VERSION)/site-packages/kllvm $(DESTDIR)$(PREFIX)/lib/$(PYTHON_DEB_VERSION)/dist-packages/kllvm
```

Comment:
Should we be following more what the 3rd line here does? For debian packages, the python versioning scheme is different. We could also just change the 3rd line to linke to `/local/lib` instead of `/lib`.

Reply:
I believe the issue is that you can't have anything in `/usr/local`, so you can't link the debian python directory to the first location.